src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -102,6 +102,12 @@
   [context]
   (keyword (get-in context [::json-body :desired_state])))
 
+(defn merge-request-settings
```

Comment:
Do we imagine in the future adding other settings that might need to be merged in? If we do, I like the generic name of this.

But if we don't think there will/should be others, or if we'd rather set ourselves up to follow a pattern where specific settings are merged in specific contexts (as opposed to all of them all the time), it might make more sense to make this more specific to just overriding the TTL.

I don't have a strong opinion either way right now though.

Reply:
I thought about making this more similar to `get-desired-state`, but I opted to make it more general because it did seem like there would be things in the future that we might want to allow.